test/uk/gov/hmrc/hec/repos/MongoSupportSpec.scala

```diff
@@ -22,10 +22,8 @@ import uk.gov.hmrc.mongo.test.MongoSupport
 
 trait MongoSupportSpec extends MongoSupport with BeforeAndAfterEach with BeforeAndAfterAll { this: Suite with Matchers ⇒
 
-  abstract override def beforeEach(): Unit = {
-    super.beforeEach()
```

Comment:
if we're definitely sure that there's no `super` behaviour we could also get rid of `super.AfterAll` as well - may be safer to leave it in though just in case

Reply:
I think we can remove it, 
 `protected def afterAll() = ()`. what do you think?

Reply:
sorry, this is the afterAll in MongoSupport
 updateIndexPreference(requireIndexedQueryServerDefault).futureValue
    super.afterAll()

Reply:
ah ok, so there's stuff in `super` in `afterAll` we need to keep then it seems - just to be safe it might be better to put `super.beforeEach` in as well in case they add behaviour for that as well that we need to be bringing in

Reply:
beforeEach is calling drop database only, so i think i can just keep `super.beforeEach` and remove `dropDatabase`()

Reply:
having double checked I think you need to use `CleanMongoCollectionSupport` instead of just `MondoSupport` to get the `dropDatabase` behaviour in `beforeEach`